node/core/prospective-parachains/src/tests.rs

```diff
@@ -15,7 +15,7 @@
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
 use super::*;
-use ::polkadot_primitives_test_helpers::{dummy_candidate_receipt_bad_sig, dummy_hash};
+use ::polkadot_primitives_test_helpers::make_candidate;
```

Comment:
Leading `::` is very 2018-era Rust - not sure why that was there, but it should be replaced